### ISSUE TITLE
Option to disable image preview scrolling.

### ISSUE THREAD

**Issue by snail-coupe** [cheeaun/phanpy#452]
Occasionally I notice images in Phanpy "scrolling". This happens both on Preview Cards for links and images that have been truncated for display on the timeline.

I _think_ this happens when the post/image has the focus (although I'm not really sure what that means on mobile!)

It's a really neat feature and I like it at a technical/UX level but... I find animations really visually distracting and so, personally, I'd rather not have them 

**Is your feature request related to a problem? Please describe.**
This does not relate to a specific problem.

However as above I (and I understand this to be a not uncommon issue) find animations overly distracting.

**Describe the solution you'd like**
An option in the settings menu that would disable the automated "scrolling" that happens to images that have the focus.

**Describe alternatives you've considered**
The only other option I can think of is disabling it completely but obviously that isn't really an acceptable one especially given I recognise this is primarily a person project.

Or not implementing and just putting up with it I guess.

**Additional context**

I really like Phanpy and it's now my default app. I hope this request makes sense because I really want sure on some of the terminology and if my assumption of when this happens is correct.

**Comment by sloanlance** [comment]
Yes, please!  I came to this repo to open an issue to request this.  I'm happy to see that somebody else has already done that.

The scrolling of images upon mouseover is annoying to say the least, and distracting to be fully honest.  Animation of this kind can be a problem for neurodivergent users, causing an accessibility issue.
